superset/db_engine_specs/base.py

```diff
@@ -454,55 +452,27 @@ def df_to_sql(cls, df: pd.DataFrame, **kwargs: Any) -> None:
         df.to_sql(**kwargs)
 
     @classmethod
-    def create_table_from_csv(cls, form: Form, database: "Database") -> None:
+    def create_table_from_csv(  # pylint: disable=too-many-arguments
+        cls,
+        filename: str,
+        table_name: str,
```

Comment:
This could be a good candidate for the [Table](https://github.com/apache/incubator-superset/blob/3cc540019f2aa6c3dac1d356f0f21eeca96b34f2/superset/sql_parse.py#L56) class (which may also remove the need for the `pylint` override).

Reply:
done